internal/pkg/service/buffer/api/service/service.go

```diff
@@ -115,6 +115,7 @@ func (*service) CreateReceiver(d dependencies.ForProjectRequest, payload *buffer
 		}
 
 		mapping := model.Mapping{
+			RevisionID:  1,
```

Comment:
Revision bude vzdy `>0`, aby sme odlisili validnu hodnotu, od toho, ze sa to zabudlo nastavit.